tools/eslint-rules/unescaped-regexp-dot.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @fileoverview Look for unescaped "literal" dots in regular expressions
+ * @author Brian White
+ */
+'use strict';
+
+const path = require('path');
+const utilsPath = path.join(__dirname, '..', 'eslint', 'lib', 'ast-utils.js');
+const astUtils = require(utilsPath);
+const getLocationFromRangeIndex = astUtils.getLocationFromRangeIndex;
+const getRangeIndexFromLocation = astUtils.getRangeIndexFromLocation;
```

Comment:
These methods aren't part of the public API, and were removed in v3.17.0. (Using them for now is probably fine, but they'll need to be changed to `sourceCode.getLocFromIndex` and `sourceCode.getIndexFromLoc` when ESLint is upgraded.)